test/e2e/framework/ingress/ingress_utils.go

```diff
@@ -831,11 +831,23 @@ type NginxIngressController struct {
 	rc         *v1.ReplicationController
 	pod        *v1.Pod
 	Client     clientset.Interface
-	externalIP string
 }
 
 // Init initializes the NginxIngressController
 func (cont *NginxIngressController) Init() {
+	framework.Logf("Creating load balancer service for nginx ingress controller")
```

Comment:
Add a comment why you do that (i.e. to make this work also on clusters in which nodes doesn't have public IPs).

Reply:
Done. Added a comment.